### PR TITLE
Install WearableData interface

### PR DESCRIPTION
**Pull request by RiccardoGrieco**
WearableData messages interface is not currently installed. This PR to install them as done for the WearableActuators.

However, this is *not* the correct way to install the files, since they are under `${CMAKE_INSTALL_PREFIX}/thrifts`.

I will open an issue for that.

### CODE REVIEW

**Review by lrapetti — APPROVED** [review]
Thanks @RiccardoGrieco

**Comment by lrapetti** [comment]
We wait for the CI before merging

> However, this is not the correct way to install the files, since they are under ${CMAKE_INSTALL_PREFIX}/thrifts.
> 
> I will open an issue for that.

we should remember to do this

**Comment by RiccardoGrieco** [comment]
cc @CarlottaSartore

**Comment by lrapetti** [comment]
I have fixed the conflicts with the master, waiting for the CI before merging

**Pull request merged**
